src/read_csv.py

```diff
@@ -1,11 +1,15 @@
 import csv
-countylist = []
-newspaperlist = []
+import sys
+import os
+
+
 def readFile(file):
+    countylist = []
+    newspaperlist = []
     with open(file, newline='') as csvfile:
         reader = csv.DictReader(csvfile)
         for row in reader:
             countylist.append(row['counties'])
             newspaperlist.append(row['localnewspapers'])
-    totallist = (countylist, newspaperlist)
+    totallist = (countylist,newspaperlist)
     return totallist

```

Comment:
Here you can just put `return countylist, newspaperlist` and remove the intermediate variable